schemas/io.snowplow.ecomm/cart/jsonschema/1-0-0

```diff
@@ -0,0 +1,166 @@
+{
+    "description": "The cart entity contains the products and bundles that have been added to cart, as well as information about discounts and total costs",
+    "properties": {
+        "cart_id": {
+            "type": "string",
+            "maxLength": 255
+        },
+        "total_items": {
+            "type": [
+                "integer",
+                "null"
+            ],
+            "description": "Total number of items in the cart.",
+            "minimum": 0,
+            "maximum": 2147483647
+        },
+        "subtotal": {
+            "type": "number",
+            "description": "The amount total for the items/products in the order, before discount is applied.",
+            "minimum": 0,
+            "maximum": 9999999
+        },
+        "shipping_cost": {
+            "type": [
+                "number",
+                "null"
+            ],
+            "description": "The total cost of shipping.",
+            "minimum": 0,
+            "maximum": 9999999
+        },
+        "discounts": {
+            "type": [
+                "array",
+                "null"
+            ],
+            "description": "Array of discount entity IDs, these are the discount(s) that has been applied to the cart.",
+            "items": {
+                "type": [
+                    "string"
+                ],
+                "maxLength": 255
+            }
+        },
+        "tax": {
+            "type": [
+                "number",
+                "null"
+            ],
+            "description": "The total tax value.",
+            "minimum": 0,
+            "maximum": 9999999
+        },
+        "total": {
+            "type": "number",
+            "description": "The total amount that is due this includes the tax, shipping and other fees; discount is deducted if applicable.",
+            "minimum": 0,
+            "maximum": 9999999
+        },
+        "currency": {
+            "type": [
+                "string",
+                "null"
+            ],
+            "description": "The ISO 4217 alphabetic currency code for which the monetary value of the cart is in.",
+            "maxLength": 3
+        },
+        "coupon_ids": {
+            "type": [
+                "array",
+                "null"
+            ],
+            "description": "IDs of coupon entities that have been attached to the cart.",
+            "items": {
+                "type": "string",
+                "maxLength": 255
+            }
+        },
+        "items_in_cart": {
```

Comment:
You can use a local $ref for both items_in_cart and discovery. Might make this a bit more readable.